src/content-tooltip/components/container.jsx

```diff
@@ -27,10 +35,56 @@ class TooltipContainer extends React.Component {
         showTooltip: false,
         position: { x: 250, y: 200 },
         tooltipState: 'copied',
+        highlightsOn: false,
     }
 
     componentDidMount() {
         this.props.onInit(this.showTooltip)
+
+        const fetchAndHighlightAnnotations = async () => {
+            const annotations = await remoteFunction('getAllAnnotations')(
+                window.location.href,
+            )
+            const highlightables = annotations.filter(
+                annotation => annotation.selector,
+            )
+            highlightAnnotations(highlightables)
+        }
+
+        const toggleHighlights = () => {
+            this.state.highlightsOn
+                ? removeHighlights()
+                : fetchAndHighlightAnnotations()
+            this.setState({ highlightsOn: !this.state.highlightsOn })
+        }
+
+        Mousetrap.bind(['r', 'h', 'a', 'l'], e => {
```

Comment:
Let's move most of this code outside of `componentDidMount`. The callback passed to `Mousetrap.bind`, for example, would be nice as a separate method, but leave the actual `Mousetrap.bind` set-up call in here.

Reply:
I moved all the helper functions to separate methods. But I thought of leaving the event handler in `componentDidMount` as otherwise I will have to clutter up the state with all the settings variables. Also since its a global keyboard shortcut handler, it might not be reusable.